about.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About Us</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://unicons.iconscout.com/release/v4.0.0/css/line.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <section class="about-container">
+      <div class="social-links">
+        <a href="#"><i class="uil uil-facebook"></i></a>
+        <a href="#"><i class="uil uil-twitter-alt"></i></a>
+        <a href="#">English</a>
+        <a href="#">Page</a>
+        <a href="#">Logout</a>
+      </div>
+      <div class="hamburger">
+        <button id="hamburger"><i class="uil uil-bars"></i></button>
+        <div class="nav-menu">
+          <div class="desktop-logo">
+            <a href="./index.html"
+              ><img src="./images/white-logo1.jpg" alt="live-concerts-logo"
+            /></a>
+          </div>
+          <div class="menu-list">
+            <img
+              class="close"
+              src="./images/menu-cross.png"
+              alt="close button"
+            />
+            <ul class="about-menu">
+              <li class="close"><a href="./about.html">About</a></li>
+              <li class="close"><a href="#events">Events</a></li>
+              <li class="close"><a href="#">Join</a></li>
+              <li class="close"><a href="#">Sponsor</a></li>
+              <li class="close"><a href="#">News</a></li>
+              <li class="book-now close"><a href="#"> Book Now</a></li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+      <main class="about-main">
+        <div class="about-title">
+          <h2>"South Africa's best marketplace for live events"</h2>
+          <h1>LIVE CONCERTS MUSIC AT THE LAKE 2022</h1>
```

Comment:
- [ ] Please increase the font size of this heading like in the original design.
### Your design
![image](https://user-images.githubusercontent.com/45460192/198296423-be503317-aa06-41f3-8e50-902ab2a1e3e9.png)
### Original
![image](https://user-images.githubusercontent.com/45460192/198296486-942fb507-7d17-49b3-8f89-b775d7319d4a.png)